hstream-store/HStream/Store/Internal/LogDevice/Checkpoint.hs

```diff
@@ -108,12 +108,12 @@ ckpStoreUpdateLSNSync store customid logid sn =
   withForeignPtr store $ \store' -> do
     void $ E.throwStreamErrorIfNotOK $ c_checkpoint_store_update_lsn_sync_safe store' customid' logid sn
 
-updateMultiSequenceNumSync
+ckpStoreUpdateMultiLSNSync
   :: LDCheckpointStore
   -> CBytes
   -> Map C_LogID LSN
   -> IO ()
-updateMultiSequenceNumSync store customid sns =
+ckpStoreUpdateMultiLSNSync store customid sns =
```

Comment:
This function is deprecated as far as I know. Using the async version is preferred.